WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -497,8 +497,8 @@ private extension OrderDetailsViewController {
                 assertionFailure("Cannot reprint a shipping label because `navigationController` is nil")
```

Comment:
How about this case (`.reprintShippingLabel`) and the related reprint symbols in `OrderDetailsDataSource`?

Reply:
This case is specifically for reprinting an existing shipping label, so I think it makes sense to leave it as-is. (Order Details displays existing/already purchased shipping labels, which can then be reprinted, whereas the shipping label purchase flow allows you to print labels for the first time after purchase.)